api.ts

```diff
@@ -12092,7 +12092,7 @@ export const DefaultApiAxiosParamCreator = function (configuration?: Configurati
             }
     
             if (advancedValidation !== undefined) { 
-                localVarFormParams.append('advanced_validation', advancedValidation as any);
+                localVarFormParams.append('advanced_validation', String(advancedValidation) as any);
```

Comment:
Isn't this a boolean?

Reply:
Yeah, [it](https://documentation.onfido.com/#upload-live-photo)'s, but need to be casted to string prior to be provided to the API.

Change happened via OpenAPI Generator update from  in [PR#65](https://github.com/onfido/onfido-openapi-spec/pull/65), which include [this](https://github.com/OpenAPITools/openapi-generator/pull/18084) fix. Good to spot that out, as I've completely forgot to enable [this](https://github.com/onfido/onfido-node/blob/master/test/resources/live-photos.test.ts#L49) test.